packages/web/src/common/store/social/tracks/sagas.ts

```diff
@@ -593,20 +593,19 @@ export function* watchSetArtistPick() {
     socialActions.SET_ARTIST_PICK,
     function* (action: ReturnType<typeof socialActions.setArtistPick>) {
       yield* waitForWrite()
-      const userId = yield* select(getUserId)
+      const userId: ID | null = yield* select(getUserId)
 
-      if (userId) {
-        yield* put(
-          cacheActions.update(Kind.USERS, [
-            {
-              id: userId,
-              metadata: {
-                artist_pick_track_id: action.trackId
-              }
+      if (!userId) return
```

Comment:
functional change? skipping the rest of the function if no user id

without a user, the rest should be a no-op, but still calling this out